src/ol_infrastructure/applications/mitxonline/edxapp_policy.hcl

```diff
@@ -25,3 +25,7 @@ path "secret-mitxonline/edx-forum" {
 path "secret-mitxonline/edx-xqueue" {
   capabilities = ["read"]
 }
+
+path "secret-operations/global/mitx_wildcard_cert" {
```

Comment:
Do we want to be that specific regarding keys under the global path or should we just allow instances to read all keys under global?

Reply:
It's better to keep things as narrowly scoped as possible so that we don't accidentally put something under global that we don't want certain instances to have access to.